docs/cloud-native-security/cspm-get-started.asciidoc

```diff
@@ -0,0 +1,124 @@
+[[cspm-get-started]]
+= Get started with CSPM
+
+[discrete]
+[[cspm-overview]]
+== Overview
+
+This page explains how to get started monitoring the security posture of your cloud assets using the Cloud Security Posture Management (CSPM) integration.
+
+NOTE: This integration currently supports Amazon Web Service (AWS). Support for other major cloud service providers is coming soon.
+
+.Requirements
+[sidebar]
+--
+* The CSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+* To view posture data, you need `read` privileges for the following {es} indices: `logs-cloud_security_posture.findings_latest-*`, `logs-cloud_security_posture.scores-*`, `Logs-cloud_security_posture.findings`.
```

Comment:
Thinking out loud: would the list of indices work as a bulleted list (sub-bullets)? It's just a little hard to parse visually as a sentence, commas blend in with other monospace characters.